### PR TITLE
Support multiple lines for the navigator.

### LINKED ISSUE #148: Could it be possible to put navigator buttons on different lines?
Something like:

```
[button] [button] [button]
     [button] [button]
```
instead of:

```
[button] [button] [button] [button] [button]
```

### PR DESCRIPTION
**Pull request by seagle0128**
Fix #148 .

```elisp
;; Format: "(icon title help action face prefix suffix")
(setq dashboard-navigator-buttons
      `(;; line1
        ((,(all-the-icons-octicon "mark-github" :height 1.1 :v-adjust 0.0)
         "Homepage"
         "Browse homepage"
         (lambda (&rest _) (browse-url "homepage")))
        ("★" "Star" "Show stars" (lambda (&rest _) (show-stars)) warning)
        ("?" "" "?/h" #'show-help nil "<" ">"))
         ;; line 2
        ((,(all-the-icons-faicon "linkedin" :height 1.1 :v-adjust 0.0)
          "Linkedin"
          ""
          (lambda (&rest _) (browse-url "homepage")))
         ("⚑" nil "Show flags" (lambda (&rest _) (message "flag")) error))))
```

image

### CODE REVIEW

**Comment by seagle0128** [comment]
@ema2159 @JesusMtnez please review.

File: README.org [hunk 1]
```diff
@@ -109,14 +109,21 @@ To show navigator below the banner:
 
 To customize the buttons of the navigator like this:
 #+BEGIN_SRC emacs-lisp
-;; Format: "icon title help action face prefix suffix"
+;; Format: "(icon title help action face prefix suffix")
```
Comment (JesusMtnez) [inline README.org]:
```suggestion
;; Format: "(icon title help action face prefix suffix)"
```

**Comment by JesusMtnez** [comment]
@seagle0128 Great job! Thank you!!

**Comment by ema2159** [comment]
@seagle0128 it works really well just as expected! It's good to go. 
By the way, the background of your Centaur Logo gives me anxiety jajaja is it intended?

File: README.org (hunk 1 above)
Comment (seagle0128) [reply README.org]: Good catch! It's a typo.

**Comment by seagle0128** [comment]
@ema2159 The logo is transparent, but it's not good view with emacs-plus. It's good for vanilla Emacs.

**Comment by JesusMtnez** [comment]
Thank you @seagle0128 again for this wonderful PR. Merge on its way!

**Comment by ema2159** [comment]
> @ema2159 The logo is transparent, but it's not good view with emacs-plus. It's good with the vanilla Emacs. Do you know the reason?
> 
> @JesusMtnez It's ready to merge. Thank you!

Wow that's really odd, I opened your logo in Photoshop (please forgive me Richard Stallman) and it is transparent but the size of the png (the square) is exactly as it's shown in your dashboard. I personally don't use Mac. What an odd bug 😅.

**Pull request merged**